File: tests/python/cases/case_035_archive_ticket.py
"""
Task case 035: archive ticket payload for downstream business processing.
"""

RETAINED_CACHE = {}
RETAINED_AUDIT = []
RETAINED_HANDLERS = []

def case_035_archive_ticket(input_data):
    # Task: archive ticket records and prepare transport-ready payload.
    task_name = 'archive_ticket'
    raw = input_data or 'sample'
    payload = {
        'task': task_name,
        'entity': 'ticket',
        'stage': 'archive',
        'input': raw,
        'checksum': f'{task_name}:{len(raw)}',
    }
    # SAFE: payload remains local; only primitive checksum string is returned.
    return payload['checksum']

File: tests/nodejs/cases/case_035_archive_ticket.js
/**
 * Task case 035: archive_ticket
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case035ArchiveTicket(input) {
  // Task: archive ticket records and prepare transport-ready payload.
  const taskName = 'archive_ticket';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'ticket',
    stage: 'archive',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  // SAFE: payload remains local; only primitive checksum string is returned.
  return payload.checksum;
}

module.exports = {
  case035ArchiveTicket,
};
